lib/nightwatch-api.js

```diff
@@ -44,8 +44,10 @@ module.exports = class NightwatchApi {
   * takeScreenshot (moduleName, testName) {
     if (!this.takeScreenshotOnFailure) return
 
+    let forbiddenCharacters = /\*|\||\\|:|"|<|>|\?|\/|\+/
```

Comment:
please use const here and use a global flag. As only the first forbidden character will be replaced